### PR TITLE
Autopilot from devguide

### PR DESCRIPTION
**Pull request by hamishwillee**
This merges the autopilot section from the devguide into the user guide, fixing the links and tidying up the entry points. If this is accepted next step is to remove the docs from the devguide. 

This copied material includes translated files - BUT these have not been included in the build (ie there is no Chinese translation of the user guide). When a translation is created these will be ready.

There are also numerous minor changes to the docs, and merging of information from other places in the guide. More work will be required but this is still IMO better.

### CODE REVIEW

File: en/flight_controller/pixhawk_series.md [hunk 1]
```diff
@@ -1,76 +1,66 @@
 # Pixhawk Series
```
Comment (hamishwillee) [inline en/flight_controller/pixhawk_series.md]: @LorenzMeier  This doc needs close review.

**Comment by hamishwillee** [comment]
@LorenzMeier I've updated the licensing information. This is ready for review.

**Comment by hamishwillee** [comment]
Hi @LorenzMeier 

Can you please check these two docs:
* [Flight Controller Selection](https://github.com/hamishwillee/px4_user_guide/blob/ef8a1d4a9323deed2711106124bacff98ba37ddd/en/getting_started/flight_controller_selection.md) - the first doc people get to about autopilots.
* [Pixhawk Series](https://github.com/hamishwillee/px4_user_guide/blob/ef8a1d4a9323deed2711106124bacff98ba37ddd/en/flight_controller/pixhawk_series.md) - the overview of what pixhawk series means.

Everything else in this merge is copy from Devguide. It might be out of date, but is no worse than was there already. 

I want to merge this because I don't want to have to keep rebasing due to other changes.

Cheers

**Comment by hamishwillee** [comment]
@LorenzMeier Can you please review this as per https://github.com/PX4/px4_user_guide/pull/48#issuecomment-324239625

I'll resolve conflicts  that have appeared in the last 24 days after that.

**Comment by LorenzMeier** [comment]
Ok to move Autopilots - what else do you need in terms of feedback

**Comment by hamishwillee** [comment]
@LorenzMeier See the two docs linked in the comment: https://github.com/PX4/px4_user_guide/pull/48#issuecomment-324239625

Basically you wanted to make sure that the text on licensing pixhawk was correct.

**Comment by hamishwillee** [comment]
@LorenzMeier  To make this easier for you I split this into two. The now (very small) PR to review is #62

**Pull request closed without merging**